src/carma-1-tenth/calc_route_deviation.py

```diff
@@ -0,0 +1,123 @@
+import sys
+import rosbag2_py
+import numpy as np
+import yaml
+import tqdm
+from rosidl_runtime_py.utilities import get_message
+from rclpy.serialization import deserialize_message
+import matplotlib.pyplot as plt
+import datetime as dt
+from scipy.interpolate import make_interp_spline
+import os
+
+STEERING_TO_SERVO_OFFSET = 0.425
```

Comment:
Consider adding a comment to describe where these values come from (params file) and to indicate that they should be updated to fit the parameter values of ackermann_to_vesc

Reply:
comment added in plot_steering_rate.py, this was removed here because it was unused